Installer.cls

```diff
@@ -9,7 +9,7 @@ XData setup
   <Default Name="app" Value="irisapp" />
   <Default Name="restapp" Value="/rest-test" />
 
-  <Namespace Name="${Namespace}" Code="${Namespace}" Data="${Namespace}" Create="yes" Ensemble="no">
+  <Namespace Name="${Namespace}" Code="${Namespace}" Data="${Namespace}" Create="yes" Ensemble="1">
```

Comment:
It's a minor nit, but I'd not add the pile of mappings for interoperability if we don't have to.

Reply:
You are right, but but I don't know if we need Ensemble for analytics, if so, I can uncheck this.
And when I have time, we can clean this installer, we don't need <Default Name="restapp" Value="/rest-test" /> and co.

Reply:
I'm not sure we need the whole %Installer in the first place (having zpm as a more lightweight solution), but it's sort of part of the template for sizeable samples. By any means, Ensemble is _not_ required for analytics.